.eslintignore

```diff
@@ -1,3 +1,4 @@
+README.md
```

Comment:
Somehow the readme file at the root does not seem to be ignored when the github action runs prettier, despite being spelled out here

Reply:
Cute. We should modify .prettierignore for that I believe :)

Reply:
Oh duh! Thank you I totally missed that!

Reply:
Hm, doesn't seem like that solved it.
I'll come back to this tomorrow and set up act locally to test them instead of pushing more blind commits…